### PR TITLE
Pull request for code review - Reviewing Hello Microverse code

### PR DESCRIPTION
**Pull request by wickathou**
This pull request was created to review the structure and workflow followed while making this project. The final pull request authorizes the merger of the last feature branch, which is a revert of a wrongly submitted pull request and the merger with the main branch

### CODE REVIEW

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
👍

File: README.md [hunk 1]
```diff
@@ -1,31 +1,2 @@
...
-- LinkedIn: [LinkedIn](https://linkedin.com/in/javierjhm)
-
-## 📝 License <a name="license"></a>
-
-This project is [GNU](./LICENSE) licensed.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
\ No newline at end of file
+# hello-microverse
+Hello world to microverse
```
Comment (mukhammadrizooff) [inline README.md:2]: - [x] [REQUIRED] Good job so far! Please Implement descriptive `README` [provided by Microverse](https://github.com/microverseinc/readme-template) 🙏 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @wickathou, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Descriptive Pull Request summary ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [x] [REQUIRED] Please implement my suggestions! You forgot to install `Linter Actions`, please install them [based on the list](https://github.com/microverseinc/linters-config/tree/master/html-css) provided by `Microverse` 🙏 

image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by wickathou** [comment]
@mukhammadrizooff I had to close this PR, because I made a mess of it! I''ll remake a new PR

**Pull request merged**